### ISSUE TITLE
Latest Nano version has breaking changes.

### ISSUE THREAD

**Issue by kajabfab** [cloudant/nodejs-cloudant#416]
## Bug Description
Nano version 8.2.0 causes `couchbackup.backup(dbUrl, stream, {mode: 'shallow'}, (err) => {})` to fail. The error message is `Invalid parameters`.

## Environment details
@Cloudant/cloudant version: 2.4.3
@cloudant/couchbackup version: 2.5.1

**Comment by ricellis** [comment]
Yeah looks like Nano 8.2.0 made some breaking changes, guess it should probably have been a 9.0.
The problem reported here is specific to Couchbackup usage so transferring there.
